src/pkg/sources/remote.go

```diff
@@ -85,6 +90,16 @@ func (r *RemoteBundle) LoadPackage(dst *layout.PackagePaths, filter filters.Comp
 		}
 	}
 	addNamespaceOverrides(&pkg, r.nsOverrides)
+
+	if config.Dev {
```

Comment:
I think we have similar code in `sources/tarball.go` (same for the `ZarfInitConfig` check above), what do you think of refactoring these snippets to be in `sources/common.go`?